src/Avalonia.Diagnostics/Diagnostics/ViewModels/MainViewModel.cs

```diff
@@ -231,6 +236,11 @@ private set
             private set => RaiseAndSetIfChanged(ref _pointerOverElementName, value);
         }
 
+        public void ShowHotKeys()
+        {
+            SelectedTab = 3;
```

Comment:
A bit cludgy here, but changing `SelectedTab` is necessary so other tabs do not remain selected.